src/templates/tournaments/tournament_list.html

```diff
@@ -15,7 +15,6 @@ <h2>
         <p>Description: {{ tournament.description }}</p>
         <p>Rules: {{ tournament.rules }}</p>
         <p>Draw rules: {{ tournament.draw_rules }}</p>
-        <!-- Matches may be added later if needed -->
         <p>Winners:</p>
```

Comment:
I think we should change how winners are displayed on ChiGame. Currently, if the tournament is in progress, the tournament has no winner, so why should the list view show "Winners" at all? I would recommend hiding that attribute/ changing what it says if there are no winners to something other than an empty space until the tournament has winners.

Reply:
That is a good idea! I will make a change on this.